make/modules/jdk.jpackage/applauncherlib.ld-version-script

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
Linker script is "just in case" to guarantee that only controlled set of functions is exported from launcher lib. Current set of g++ OpenJDK command line options disables export of functions with external linkage by default. However in my local builds this was not the case and the whole C++ runtime statically linked in launcher lib got exported. This prevented it from unloading from launcher's process memory when it was `dlclose()`-ed. This linker script used in linkage of launcher lib will guarantee this will not happen.